roles/openshift_health_checker/test/fluentd_config_test.py

```diff
@@ -4,8 +4,8 @@
 
 
 @pytest.fixture
-def fluentd_config():
-    return FluentdConfig("dummy")
+def fluentd_config(execute_module, task_vars):
```

Comment:
Pytest Fixtures typically do not take arbitrary arguments...
It can take a magical `request` argument https://docs.pytest.org/en/latest/fixture.html#fixtures-can-introspect-the-requesting-test-context, or reference other fixtures https://docs.pytest.org/en/latest/fixture.html#modularity-using-fixtures-from-a-fixture-function